### PR TITLE
M3-5790: Login & Authentication Scaffolding for Security Questions and Phone Verification

### PR DESCRIPTION
**Pull request by bnussman**
## Description

- Changes `Linode Authentication` to `Security Settings`
- Adds placeholders for `Phone Verification` and `Security Questions`
- Moves all 3rd party login information to a single `Login Method` paper
- Adds dividers as shown in mock up (in internal ticket)

## How to test

- Check UI compared to the mockup

### CODE REVIEW

File: packages/manager/src/features/Profile/AuthenticationSettings/TPAProviders.tsx [hunk 1]
```diff
@@ -183,45 +184,44 @@ export const TPAProviders: React.FC<CombinedProps> = (props) => {
             );
           })}
         </Grid>
+        {isThirdPartyAuthEnabled && (
```
Comment (DevDW) [inline packages/manager/src/features/Profile/AuthenticationSettings/TPAProviders.tsx:187]: we can go with the ternary here, i.e., `isThirdPartyAuthEnabled ? ( ... ) : null`

File: packages/manager/src/features/Profile/AuthenticationSettings/AuthenticationSettings.tsx [hunk 2]
```diff
@@ -40,55 +38,65 @@ export const AuthenticationSettings: React.FC<{}> = () => {
...
+            <Typography variant="h3">Phone Verification</Typography>
+            <Typography
+              variant="body1"
+              style={{ marginTop: 8, marginBottom: 8 }}
+            >
+              This is a placeholder for the Phone Verification component.
+            </Typography>
+            <Divider spacingTop={22} spacingBottom={16} />
+            <TrustedDevices />
+            {ipAllowlisting && (
```
Comment (DevDW) [inline packages/manager/src/features/Profile/AuthenticationSettings/AuthenticationSettings.tsx:86]: can go with the ternary here as well

File: packages/manager/src/features/Profile/AuthenticationSettings/AuthenticationSettings.tsx [hunk 3]
```diff
@@ -40,55 +38,65 @@ export const AuthenticationSettings: React.FC<{}> = () => {
...
+              Security Settings
+            </Typography>
+            <Divider spacingTop={24} spacingBottom={16} />
+            <ResetPassword username={username} />
+            <Divider spacingTop={22} spacingBottom={16} />
+            <TwoFactor
+              twoFactor={twoFactor}
+              username={username}
+              clearState={clearState}
+            />
```
Comment (DevDW) [inline packages/manager/src/features/Profile/AuthenticationSettings/AuthenticationSettings.tsx:67]: TFA will be an option when a user has TPA enabled now?

**Review by DevDW — COMMENTED** [review]
looks good so far, left a couple of minor comments and a question re: TFA w/ TPA

File: packages/manager/src/features/Profile/AuthenticationSettings/AuthenticationSettings.tsx (hunk 3 above)
Comment (bnussman) [reply packages/manager/src/features/Profile/AuthenticationSettings/AuthenticationSettings.tsx:67]: Good catch. They should probably not see 2FA if they use 3ed party auth

Comment (bnussman) [reply packages/manager/src/features/Profile/AuthenticationSettings/AuthenticationSettings.tsx:85]: Fixed in 56dbde9

Comment (bnussman) [reply packages/manager/src/features/Profile/AuthenticationSettings/AuthenticationSettings.tsx:85]: This is what user's should see when they use TPA (third party authentication)

Screen Shot 2022-04-25 at 2 33 23 PM

**Review by DevDW — APPROVED** [review]
LGTM now 👌

**Review by tiffwong — APPROVED** [review]
Great job! 🚀

**Pull request merged**